src/app/pages/account/users/user-list/user-list.component.html

```diff
@@ -1,3 +1,46 @@
+<ng-template #pageHeader>
+  <div class="header-container">
+    <div class="title-container">
+      <a [routerLink]="['/', 'network']"><strong>NETWORK</strong></a>
+      <h1>Users</h1>
+    </div>
```

Comment:
Let's also simplify life for other developers in the future, so that they don't have to copy css.
For example, you can allow `<pagetitle>` to be put here. It can also be made a bit simpler (for example by defaulting `breadcrumbs` to true and nuking `productType`.